packages/ra-ui-materialui/src/preferences/Configurable.tsx

```diff
@@ -73,6 +73,12 @@ export const Configurable = (props: ConfigurableProps) => {
     }
 
     const handleOpenEditor = () => {
+        if (!setEditor) {
+            throw new Error(
+                'PreferencesEditorContext is not available. Did you forget to put a PreferencesEditorContextProvider at a higher level?'
```

Comment:
**suggestion**(niptick): I like the way you present this error but other errors of this type throw this message: `Configurable must be used inside a PreferencesEditorContextProvider`. Could use the same shape to stay consistent?

```suggestion
                'Configurable must be used inside a PreferencesEditorContextProvider'
```